frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -687,65 +753,7 @@ const ManagePolicyPage = ({
             </p>
           )}
         </div>
-        <div>
-          {isAnyTeamSelected && teamPoliciesError && <TableDataError />}
-          {isAnyTeamSelected &&
-            !teamPoliciesError &&
-            (isFetchingTeamPolicies ? (
```

Comment:
Root issue: can't use a spinner to replace the table level on fetch or else it will re-render (and thus unfocus) the search input field